Library/LMS.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Library
+{
+	class LMS
+	{
+      
+
+        public static void Main(String[] args)
+		{
+			Operations bookOperations = new Operations();
+			Console.WriteLine("Enter the total number of Operations to be performed ");
+			String NoOperations = Console.ReadLine();
+			int noOperations = Convert.ToInt32(NoOperations);
+			Console.WriteLine("Total Operations to be performed: " + noOperations);
+
+			while (noOperations > 0)
```

Comment:
Program will end abruptly due to this while loop. Instead maybe have an option to exit the program.

Reply:
I have made the changes in the code.